src/components/color-picker/color-picker.css

```diff
@@ -98,6 +98,7 @@
     flex-direction: row;
     justify-content: center;
     margin: 8px;
+    user-select: nonr;
```

Comment:
Thanks!
Looks like there is a typo here in 'none'

It would be good if you could run the scratch-paint playground to make sure the change is working, to catch stuff like this.

Reply:
Fixed. I'm not good at running the playground.

Reply:
Ok, let me know if you need help setting it up